source/access/managing_disk_usage.rst

```diff
@@ -1,105 +1,138 @@
-
-Each line represents a file system you have access to,$VSC_HOME, $VSC_DATA,
-and, for this particular example, $VSC_SCRATCH_SITE. The blocks     
-column shows your current usage, quota is the usage above which    
-you will be warned, and limit is "hard", i.e., when your usage  
-reaches this limit, no more information can be written to the 
-file system, and programs that try will fail.                    
-                                  
-Some file systems have limits on the number of files that can be   
-stored, and those are represented by the last four columns. The     
-number of files you currently have is listed in the column      
-files, quota and limit represent  the soft and hard limits for the  
-number of files.                  
-                                  
-Diskspace used by individual directories         
-----------------------------------------
-                                  
-The command to check the size of  all subdirectories in the current 
-directory is ``du``
-                                  
-::                                
-                                  
-   $ du -h                        
-   4.0k      ./.ssh               
-   0         ./somedata/somesubdir  
-   52.0k     ./somedata             
-   56.0k   .                      
-                                  
-                                  
+                 648M   2919M   3072M            3685       0       0
+     nas2-ib1:/mnt/data
+               20691M  24320M  25600M            134k       0       0
+     nas1-ib1:/mnt/site_scratch
+                    0  24320M  25600M               1       0       0
+
+Each line represents a file system you have access to, ``$VSC_HOME``,
+``$VSC_DATA``, and, for this particular example, ``$VSC_SCRATCH_SITE``.
+When using the ``-s`` flag, ``quota`` will report disk space and limits
+in human-readable format, i.e., using MB or GB, rather than blocks.
+
+Some file systems have limits on the number of files that can be
+stored, and those are represented by the last four columns. The
+
+``files``
+   The number of files on the file system owned by you.
+``quota``
+   If the number of files exceeds this value, you may
+   get warnings (disabled on most systems).  The reported number
+   of files will be marked with an ``*`` if it exceeds the value
+   of ``quota``.
+``limit``
+   If the limit is reached, no new files can be written to disk, so
+   applications and jobs may crash.
+
+.. note::
+
+   Using these commands on another cluster than the one
+   in your home institution, will fail to return information
+   about the quota on your ``$VSC_HOME`` and ``$VSC_DATA``
+   directories and will show you  the quota for your ``$VSC_SCRATCH``
+   directory on that system.
+
+
+.. _mmlsquota:
+
+Total disk space on GPFS file systems
+-------------------------------------
+
+On some parallel file systems, notably ``$VSC_SCRATCH`` on KU Leuven
```

Comment:
the main issue with mmlsquota and also du is that it reports used disk space, incl replicas. so this will confuse people
it's a pity that these commands are documented
also running `du` is pretty hard on the filesystem, so we would also like to avoid any reference to it
and to make things worse, du reports aggregate of file sizes, not the actual disk usage, so people will again get confused why quota usage and du do not match

Reply:
seems that du does not aggregate filesizes, it does somethign even stranger, but nonetheless, it's not the disk usage

Reply:
With respect to `du`, I agree that is not accurate and puts strain on the file system.  However, note that it is in the HPC intro documentation.  Please open this discussion there as well.

With respect to `mmlsquota`, we need some way for a user to check quota on GPFS systems, suggestions for alternatives are welcome.

Reply:
@gjbex as ward wrote below, at VUB we use a wrapper (called myquota) that does essentially this for vsc accounts:
```
function vscquota {
awk -v repli=$1 -v fileset=$2 -F: ' /USR/ { if ($23 == fileset) { current=$11; soft=$12 ; hard=$13 ; grace=$15 } }
END {
  printf "	Current data usage: %.2f GB\n", (current/repli/1024/1024)
  printf "	Maximum data usage: %.2f GB\n", (soft/repli/1024/1024)
  if (grace != "none")
    {
    print "	After", grace, "you will not be able to write more data"
    printf "	until you decrease your data usage below %.2f GB\n", (soft/repli/1024/1024)
    }
} '
}
echo "Quota for \$VSC_HOME"
/usr/lpp/mmfs/bin/mmlsquota -Y theiahome:${USER:0:6} | vscquota 2 ${USER:0:6}
echo
echo "Quota for \$VSC_DATA"
/usr/lpp/mmfs/bin/mmlsquota -Y theiadata:${USER:0:6} | vscquota 2 ${USER:0:6}
echo
echo "Quota for \$VSC_SCRATCH"
/usr/lpp/mmfs/bin/mmlsquota -Y theiascratch:${USER:0:6} | vscquota 1 ${USER:0:6}
```

Reply:
@smoors Nice, but it doesn't exist in Gent/Antwerp.  I've no issues adapting it for Leuven ($VSC_HOME/$VSC_DATA is NFS, not GPFS), but it doesn't make sense to document it unless it is part of the CUE.

@wpoely86 @stdweird I'm quite happy to throw out the part on `du`, but that means the users need a standard way of checking quota, so how?

Reply:
for ugent users, we tell them to use the AP.

Reply:
and we don't have a CLI yet (but that is mainly due to mapower issues)